tracing-subscriber/src/fmt/mod.rs

```diff
@@ -1188,10 +1188,13 @@ pub fn try_init() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
 /// If the `tracing-log` feature is enabled, this will also install
```

Comment:
maybe adding something like this:
```suggestion
/// The configuration of the subscriber initialized by this function
/// depends on what [feature flags](crate#feature-flags) are enabled.
///
/// If the `tracing-log` feature is enabled, this will also install
```